src/Common/Core/Impl/Enums/PrintOptionsEnum.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core {
+    public enum PdfPrintOptionsEnum {
```

Comment:
Unlike `OrientationEnum`, `PdfPrintOptionsEnum` is specific to the R function. So it has to be in the same assembly and namespace with the rest of the "Export to PDF" functionality. Though I'm not sure if it is required at all (see below).